Clinical-Genomics/scout
# Clear panel pending

This PR allows to cancel the pending changes to a gene panel. It also improves the badges of the pending actions (fix #1665)

**How to test**:
1. Install the branch on a local implementation of Scout, or on stage.
1. Try to modify a gene panel on the panel page, by adding, editing and deleting a gene. The relative badge should now display deleteED, addED, editED  (instead of delete, add, edit) as the pending actions for those genes:

![image](https://user-images.githubusercontent.com/28093618/73754568-dc818980-4764-11ea-9aa9-ee5f6eefe454.png)

1. Try to cancel those pending actions by clicking on the `Cancel pending actions` button and the pending list of actions should be cleared, and gene panel version should stay the same.
![image](https://user-images.githubusercontent.com/28093618/73754679-08047400-4765-11ea-9b2f-be9b88e6a09f.png)

**Expected outcome**:
The functionality should be working

**Review:**
